sass/variables/_colors.scss

```diff
@@ -1,3 +1,10 @@
+/* Colors for Contact-us & About-us pages */
+$color__background-contact-about: #2fbeb6;
+$color__text-title-contact-about: #FFF;
+$color__text-content-contact-about: #404041;
```

Comment:
On Second thought I like the idea of starting them with color. Let's chat about this tuesday